### PR TITLE
Add New Beginnings Art Contest Results news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎆

### CODE REVIEW

File: news/2022-12-31-new-beginnings-results.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+![](https://assets.ppy.sh/contests/160/banners/Dreamxiety.jpg)
+
+Throughout time immemorial, some people have proudly and grandiosely claimed that 'every end is a new beginning'. And so, as 2022 fades into memory (thank goodness), we stand on the precipice of a new and exciting year with everything that brings.
+
+Which in this case, is a new set of Art Contest finalists to start with!
+
+Thanks to the exhaustive efforts of **4273** unique voters, **19** finalists have been decided, with 15 from the initial voting and an extra 4 as chosen by the CEO of circles himself.
+
+Let's get stuck straight into things - and remember, you can click any banner to be taken to a full-size version of the art in all of its glorious quality.
```
Comment (0x84f) [inline news/2022-12-31-new-beginnings-results.md:17]: I _think_ this is correct. If that's the case, Wala sends their regards.

```suggestion
Let's get stuck straight into things — and remember, you can click any banner to be taken to a full-size version of the art in all of its glorious quality.
```

File: news/2022-12-31-new-beginnings-results.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+[![](https://assets.ppy.sh/contests/160/banners/Dreamxiety.jpg)](https://assets.ppy.sh/contests/160/winners/Dreamxiety.png)
+
+Welcoming in 2023 with  **2203** votes (we can't make this stuff up, honestly) is [**Dreamxiety**](https://osu.ppy.sh/users/13103233)'s rendition of a kimono-clad pippi enjoying a sparkler amidst a sea of stars and floating lanterns!
```
Comment (0x84f) [inline news/2022-12-31-new-beginnings-results.md:21]: dooble space

```suggestion
Welcoming in 2023 with **2203** votes (we can't make this stuff up, honestly) is [**Dreamxiety**](https://osu.ppy.sh/users/13103233)'s rendition of a kimono-clad pippi enjoying a sparkler amidst a sea of stars and floating lanterns!
```

File: news/2022-12-31-new-beginnings-results.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Here's the official count of Dreamxiety's wins so far: *many*. And with a style like this, it's not hard to see why. Well done!
+
+[![](https://assets.ppy.sh/contests/160/banners/torekka.jpg)](https://assets.ppy.sh/contests/160/winners/torekka.png)
+
+**2045** voters stumbled into [**torekka**](https://osu.ppy.sh/users/25130329) capturing a candid moment between pippi and Mocha, all set to a backdrop of fantastical fireworks. Getting swept up in things is definitely on brand for these two.
+
+[![](https://assets.ppy.sh/contests/160/banners/Nimiru.jpg)](https://assets.ppy.sh/contests/160/winners/Nimiru.png)
+
+The new year festivities bloom in full with [**Nimiru**](https://osu.ppy.sh/users/11041959)'s take on a gorgeous Eastern-themed celebration.  **2029** voters approved.
```
Comment (0x84f) [inline news/2022-12-31-new-beginnings-results.md:31]: dooble space

```suggestion
The new year festivities bloom in full with [**Nimiru**](https://osu.ppy.sh/users/11041959)'s take on a gorgeous Eastern-themed celebration. **2029** voters approved.
```

File: news/2022-12-31-new-beginnings-results.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+Simply glorious!
+
+[![](https://assets.ppy.sh/contests/160/banners/Nemururin.jpg)](https://assets.ppy.sh/contests/160/winners/Nemururin.png)
+
+**1912** voters joined in the festive firework awe with [**Nemururin**](https://osu.ppy.sh/users/26391825)'s rendition of pippi, Mocha, Yuzu and a don enjoying the scenery.
+
+[![](https://assets.ppy.sh/contests/160/banners/Son_Goku.jpg)](https://assets.ppy.sh/contests/160/winners/Son%20Goku.png)
+
+A crowd of **1693** voters joined [**Son Goku**](https://osu.ppy.sh/users/14991975)'s pippi, Mocha, Chirou and a don in perusing through some festive stalls for future fortunes. Well, two of them, at least. Mocha's rocking a sparkler with mischief in her eyes and the don beside her clearly knows what's up.
```
Comment (0x84f) [inline news/2022-12-31-new-beginnings-results.md:57]: subjective

```suggestion
A crowd of **1693** voters joined [**Son Goku**](https://osu.ppy.sh/users/14991975)'s pippi, Mocha, Chirou and a don in perusing through some festive stalls for future fortunes. Well, two of them at least. Mocha's rocking a sparkler with mischief in her eyes and the don beside her clearly knows what's up.
```

File: news/2022-12-31-new-beginnings-results.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+[![](https://assets.ppy.sh/contests/160/banners/ksenikust.jpg)](https://assets.ppy.sh/contests/160/winners/ksenikust.png)
+
+[**ksenikust**](https://osu.ppy.sh/users/8938068) goes clean, crisp and retro-future in their rendition of an insanely stylish pippi to the approval of **1660** voters. The colour work on this piece is phenomenal. Absolutely outstanding work!
+
+[![](https://assets.ppy.sh/contests/160/banners/RedcXca.jpg)](https://assets.ppy.sh/contests/160/winners/RedcXca.png)
+
+[**RedcXca**](https://osu.ppy.sh/users/14056601) decrees that Alisa's time to shine is right now, and  **1404** voters agreed. Her base outfit really does fit the occasion, doesn't it?
```
Comment (0x84f) [inline news/2022-12-31-new-beginnings-results.md:65]: dooble space

```suggestion
[**RedcXca**](https://osu.ppy.sh/users/14056601) decrees that Alisa's time to shine is right now, and **1404** voters agreed. Her base outfit really does fit the occasion, doesn't it?
```

**Review by peppy — APPROVED** [review]

**Pull request merged**